regression/cbmc-java/array_ref_multidim_nonconstsize_nonconstaccess_2/test.desc

```diff
@@ -0,0 +1,12 @@
+KNOWNBUG
```

Comment:
Is this definitely knownbug? I thought second dimension being variable was fine?

Reply:
As it turns out, second dimension variation of the example works fine for `float[][]` but not with references `A[][]`.

Reply:
How odd...